server/src/auth/get-user.decorator.ts

```diff
@@ -0,0 +1,7 @@
+import { createParamDecorator, ExecutionContext } from "@nestjs/common";
+import { User } from "src/users/entities/user.entity";
+
+export const GetUser = createParamDecorator((_data, ctx: ExecutionContext): User => {
+  const req = ctx.switchToHttp().getRequest()
+  return req.user
+})
```

Comment:
nice! much more nesty than `request.user = user;` in https://github.com/Nonprofit-Exchange-Hub/web-app/pull/91/files